site/_guides/cert-manager.md

```diff
@@ -487,11 +505,108 @@ $ curl https://httpbin.davecheney.com/get
+This works because Contour expects the Secrets to look the same anyway.
+
+To do this, we need to create our HTTPProxy and Ingress objects.
+
+This example uses the hostname `httpbinproxy.davecheney.com`, remember to create that name before starting.
+
+Firstly, the HTTPProxy:
+
+```yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: httpbinproxy
+spec:
+  virtualhost:
+    fqdn: httpbinproxy.davecheney.com
+    tls:
+      secretName: httpbin
+  routes:
+  - services:
+    - name: httpbin
+      port: 8080
+
+```
+
+This config will be marked as Invalid by Contour, since the TLS secret doesn't exist yet.
+Once that's done, create the dummy Ingress object:
+
+```yaml
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.allow-http: "false"
+    kubernetes.io/ingress.class: contour
+    kubernetes.io/tls-acme: "true"
+  name: httpbinproxy
+  namespace: default
+spec:
+  rules:
+  - host: httpbinproxy.davecheney.com
+    http:
+      paths:
+      - backend:
+          serviceName: httpbin
+          servicePort: 8080
+  tls:
+  - hosts:
+    - httpbinproxy.davecheney.com
+    secretName: httpbinproxy
+```
+
+Once cert-manager has done its thing, you will have a `kubernetes.io/tls` Secret present, that will contain the keypair.
+Contour will detect that the Secret exists and generate the HTTPProxy config.
+
+After that, you should be able to curl the new site:
+
+```sh
+$ curl https://httpbinproxy.davecheney.com/get
+{
+  "args": {},
+  "headers": {
+    "Accept": "*/*",
+    "Content-Length": "0",
+    "Host": "httpbinproxy.davecheney.com",
+    "User-Agent": "curl/7.54.0",
+    "X-Envoy-Expected-Rq-Timeout-Ms": "15000",
+    "X-Envoy-External-Address": "122.106.57.183"
+  },
+  "origin": "122.106.57.183",
+  "url": "https://httpbinproxy.davecheney.com/get"
+}
+```
+
+### Caveats
+
+This method is a workaround until we can deliver the changes in [#950](https://github.com/projectcontour/contour/issues/950) and [#951](https://github.com/projectcontour/contour/issues/951).
+As such, it has a few caveats.
+
+Firstly, it works because Contour will override matching FQDN/path combinations in the following order: HTTPProxy overrides IngressRoute overrides Ingress.
+The solver Ingress created by cert-manager has a unique FQDN/path combination so it will work, but the base one will be overridden by the HTTPProxy config.
+
+Secondly, the actual config of where the object sends traffic is irrelevant, *as long as it matches a route in the HTTPProxy*.
+In this example, we're not using any advanced routing features, so this is straightforward.
+However, if you are, you'll either need to match routes between Ingress and HTTPProxy or create a dummy path for the Ingress to match on.
```

Comment:
I think this could be said simpler by saying the ingress record exists for ingress-shim only, mixing routes across ingress and httpproxy is outside the scope of this document.